parasolr/query/queryset.py

```diff
@@ -72,16 +74,42 @@ def get_results(self, **kwargs) -> List[dict]:
         # if query options have changed?
         # For now, always query.
 
+        # if cached and no override query args are specified,
+        # return existing cached result
+        if self._result_cache and not kwargs:
+            return self._result_cache
+
         query_opts = self.query_opts()
         query_opts.update(**kwargs)
+
         # TODO: what do we do about the fact that Solr defaults
         # to 10 rows?
```

Comment:
Hm– this parameter can still be changed with the kwargs right? If so I think it's ok to leave this out.

Reply:
We've had this TODO in here so long that I've stopped seeing it; I think we have an issue for the problem but I'm not currently sure the best solution. I think we can continue to live with it, but I'll update the comment to make it a NOTE and reference the ticket.